pkgs/applications/terminal-emulators/rio/default.nix

```diff
@@ -112,7 +112,11 @@ rustPlatform.buildRustPackage rec {
     description = "A hardware-accelerated GPU terminal emulator powered by WebGPU";
     homepage = "https://raphamorim.io/rio";
     license = lib.licenses.mit;
-    maintainers = with lib.maintainers; [ otavio oluceps ];
+    maintainers = with lib.maintainers; [
```

Comment:
*split*

Reply:
I... don't understand what you mean. I though with "split" you meant "split the list into several lines".